charts/portworx/v1.0.0/templates/portworx-lighthouse.yaml

```diff
@@ -11,9 +11,18 @@ metadata:
    name: px-lh-role
    namespace: kube-system
 rules:
-- apiGroups: [""]
-  resources: ["configmaps"]
-  verbs: ["get", "create", "update"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "create", "update"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "create", "update"]
+  - apiGroups: [""]
+    resources: ["nodes", "services"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["stork.libopenstorage.org"]
+    resources: ["clusterpairs"]
```

Comment:
this needs "migrations" and "groupvolumesnapshots" aswell.

Reply:
Also this needs to become ClusterRole and ClusterRoleBinding see latest spec form install.portworx.com/2.0